libs/gltfio/src/AssetLoader.cpp

```diff
@@ -535,8 +536,8 @@ void FAssetLoader::createRenderable(const cgltf_data* srcAsset, const cgltf_node
     // node weights are provided, they override the ones specified on the mesh.
     if (numMorphTargets > 0) {
         RenderableManager::Instance renderable = mRenderableManager.getInstance(entity);
-        const auto size = std::min(MAX_MORPH_TARGETS, std::max(mesh->weights_count, node->weights_count));
-        std::vector<float> weights(size);
+        const auto size = std::min(MAX_MORPH_TARGETS, numMorphTargets);
+        std::vector<float> weights(size, 0.0f);
```

Comment:
I realize this is pre-existing, but as a cleanup can we change this to FixedCapacityVector?  std::vector isn't used anywhere else in this file.  In fact I would also like SkinVector to also be FixedCapacityVector.

Reply:
It's updated.